includes/helpers/class-app.php

```diff
@@ -345,6 +345,21 @@ public static function kses( $value, $allowed = array() ) {
 		return wp_kses( $value, $allowed_html );
 	}
 
+	/**
+	 * Check if font awesome is enabled.
+	 * This checks if the plugin is active or the styles are loaded.
+	 *
+	 * @since x.x
+	 *
+	 * @return bool
+	 */
+	public static function is_font_awesome_enabled() {
+		if ( is_plugin_active( 'font-awesome/index.php' ) || wp_style_is( 'font-awesome', 'enqueued' ) ) {
```

Comment:
Can we slim this down to:
`return is_plugin_active( 'font-awesome/index.php' ) || wp_style_is( 'font-awesome', 'enqueued' );`